src/scenes/CreateEvent/index.js

```diff
@@ -378,8 +398,54 @@ export default class CreateEvent extends Component {
     if (nextProps.txReturn) {
       this.onClose();
     }
+
+    if (!_.isUndefined(nextProps.addressValidated)) {
+      this.renderResultSetter();
+    }
   }
 
+  renderResultSetter = () => {
```

Comment:
change to stateless component, like `resultSetterTextField`. and rename this to `resultSetterField` or something like that.